src/halfpipe/tui/base.py

```diff
@@ -0,0 +1,254 @@
+        background: $foreground 10%;
+    }
+    """
+    """The character to use as the icon within the header."""
+    icon = Reactive("❓")
+    help_string = "Here should be some general help :) Or maybe link to manual?"
+
+    async def on_click(self, event: Click) -> None:
+        """Launch the command palette when icon is clicked."""
+        event.stop()
+        await self.app.push_screen(
+            Confirm(
+                self.help_string,
+                left_button_text=False,
+                right_button_text="OK",
+                right_button_variant="default",
+                title="Help",
+                id="help_modal",
+                #   classes="confirm_warning",
+            )
+        )
+
+    def render(self) -> RenderResult:
+        """Render the header icon.
+
+        Returns:
+            The rendered icon.
+        """
+        return self.icon
+
+
+class MyHeader(Header):
+    def compose(self):
+        yield HeaderTitle()
+        yield HeaderHelpIcon()
+        yield HeaderCloseIcon()
+
+
+class RichImage:
+    """Convert the image to a Rich image."""
+
+    # def __rich_console__(self, console: Console, options) -> RenderResult:
+    #     with Image.open(os.path.join(BASE_DIR, "images/halfpipe_logo_v2.png")) as image:
+    #         pixels = Pixels.from_image(image, resize=(110, 92))  # 105, 92
+    #     return console.render(pixels)
+
+
+class ImageContainer(Container):
+    """Create a container for the image."""
+
+    def __init__(self, **kwargs) -> None:
+        super().__init__(**kwargs)
+
+    def render(self):
+        return RichImage()
+
+    def key_escape(self):
+        self.dismiss(False)
+
+
+class Welcome(ModalScreen):
+    """Intro screen with an intro image."""
+
+    def __init__(self, **kwargs) -> None:
+        super().__init__(**kwargs)
+
+    def compose(self) -> ComposeResult:
+        yield ImageContainer(id="welcome_image")
+
+    def key_escape(self):
+        self.dismiss(True)
+
+    def on_click(self, event: events.Click) -> None:
+        self.dismiss(True)
+
+
+class MainApp(App):
+    CSS_PATH = [
+        os.path.join(BASE_DIR, "tcss/base.tcss"),
+        os.path.join(BASE_DIR, "feature_widgets/tcss/base.tcss"),
+        os.path.join(BASE_DIR, "feature_widgets/tcss/taskbased.tcss"),
+        os.path.join(BASE_DIR, "feature_widgets/tcss/model_conditions_and_contrasts.tcss"),
+        os.path.join(BASE_DIR, "utils/tcss/file_browser.tcss"),
+        os.path.join(BASE_DIR, "working_directory/tcss/working_directory.tcss"),
+        os.path.join(BASE_DIR, "data_input/tcss/data_input.tcss"),
+        os.path.join(BASE_DIR, "preprocessing/tcss/preprocessing.tcss"),
+        os.path.join(BASE_DIR, "run/tcss/run.tcss"),
+        os.path.join(BASE_DIR, "utils/tcss/path_pattern_builder.tcss"),
+        os.path.join(BASE_DIR, "dev.tcss"),
+        os.path.join(BASE_DIR, "utils/tcss/radio_set_changed.tcss"),
+    ]
+
+    # TODO: The non active tabs should not show the bindings.
```

Comment:
It might be good to go over the `TODO` comments and remove them if they're no longer relevant